tests/src/Common/test_runtime/project.json

```diff
@@ -2,9 +2,6 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "2.0.0-preview1-25210-01",
     "Microsoft.DotNet.CoreCLR.TestDependencies": "1.0.0-prerelease",
-    "jit-dasm": "0.0.1.4",
-    "cijobs": "0.0.1.2",
-    "jit-analyze": "0.0.1.1"
```

Comment:
This appears to be because these packages depend on System.CommandLine from [corefxlab](https://dotnet.myget.org/feed/dotnet-corefxlab/package/nuget/System.CommandLine), but restore is getting a conflicting package with the same name from [nuget](https://www.nuget.org/packages/System.CommandLine/). The solution is to specify corefxlab as a source during the restore of these packages. On the command line you can do that by passing --source, but in our test builds there's probably a different place to do this, maybe in a NuGet.config. @ramarag, do you know where the restore sources are specified for this project.json?